lib/active_record_host_pool/pool_proxy.rb

```diff
@@ -150,7 +150,8 @@ def _connection_proxy_for(connection, database)
 
       @connection_proxy_cache[key] ||= begin
         cx = ActiveRecordHostPool::ConnectionProxy.new(connection, database)
-        cx.execute("select 1")
```

Comment:
Hmm, I believe removing this line means that calling `_connection_proxy_for` will no longer switch databases. I need to think if there are any scenarios where that might cause problems for us.